.moban.yaml

```
@@ -28,8 +28,9 @@ entry_points:
     - coala-quickstart = coala_quickstart.coala_quickstart:main
 
 dependencies:
-  - coala_bears~=0.12.0.dev20170722110839
-  - coala_utils~=0.6.6
+  # PR 5706, 7806, and 5809
+  - https://github.com/coala/coala/archive/e4e780de.zip
+  - git+https://github.com/coala/coala-bears
```

What is the use of `git+https` when simply
```suggestion
  - https://github.com/coala/coala-bears/archive/master.zip
```
would work?

---

Because using git is faster; the repo can be cached by pip , and master is refetched, which is a tiny operation, as opposed to fetching the entire set of files.